requirements.txt

```diff
@@ -1,2 +1,3 @@
 Jinja2==2.8
+Django==1.11.4
```

Comment:
Maybe we can leave Django out of the requirements and let the users install Django only if they want to use the DjangoRenderer. What do you think?